pkg/apis/addtoscheme_operator_v1.go

```diff
@@ -15,8 +15,8 @@
 package apis
 
 import (
-	esv1 "github.com/elastic/cloud-on-k8s/pkg/apis/elasticsearch/v1"
-	kbv1 "github.com/elastic/cloud-on-k8s/pkg/apis/kibana/v1"
+	esv1 "github.com/elastic/cloud-on-k8s/v2/pkg/apis/elasticsearch/v1"
```

Comment:
I can live with it, but it's a bit weird to have the v2 api imported as esv1

Reply:
`/v2/.../v1/` 

What a crazy world we live in.

Reply:
Yeah, @pasanw explained to me that I didn't read it carefully enough and the CRDs are still v1...